### ISSUE TITLE
Deleting a data source not working correctly

### ISSUE THREAD

**Issue by PRijnbeek** [EHDEN/ARACHNE#15]
When i delete my data source i cannot added with the same name later. looks like it is not really deleted?

![image](https://user-images.githubusercontent.com/7252105/118128944-d7959600-b3fb-11eb-9a28-d0b16ef8a264.png)

**Comment by konstjar** [comment]
The datasource is not removed physically. It's just mark as deleted in system and hidden. It's required to keep the reference for other entries in the application. But I agree, the name should not be a problem.

**Comment by PRijnbeek** [comment]
Yes it should not be the primary key in the table but a hash or something like that
